page-h5p-listing.php

```diff
@@ -3,40 +3,65 @@
 <?php
 if ( \PressbooksBook\Helpers\is_book_public() ) :
 	$data = \PressbooksBook\Helpers\get_h5p_activities();
+	$activities = count( $data['activities'] );
+	$filtered_by_id_exists = isset( $_GET['h5p_id'] ) && $activities === 1;
+	$filtered_does_not_exist = isset( $_GET['h5p_id'] ) && $activities === 0;
 	?>
 	<div id="post-<?php the_ID(); ?>" <?php post_class( 'h5p-listing-page' ); ?>>
-			<h1 class="page-title"><?php echo __( 'H5P activities list', 'pressbooks-book' ); ?></h1>
+		<h1 class="page-title">
+			<?php
+			if ( $filtered_by_id_exists ) {
+				echo $data['activities'][0]['title'];
+			} elseif ( $filtered_does_not_exist ) {
+				echo __( 'Invalid H5P activity', 'pressbooks-book' );
```

Comment:
@SteelWagstaff do you consider this title is appropriate for invalid H5p_id parameters?

Reply:
Yes, that's fine.